xpdan/utils.py

```diff
@@ -20,7 +20,7 @@ def _clean_info(input_str):
 def _timestampstr(timestamp):
     """ convert timestamp to strftime formate """
     timestring = datetime.datetime.fromtimestamp(float(timestamp)).strftime(
-        '%Y%m%d-%H%M')
+            '%Y%m%d-%H%M:%S')
```

Comment:
good point. I checked,  it isn't. I think we might want to use ``-`` then? 

``_`` has been used to separate different parts of information